<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nomadic Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>39</y>
      <width>37</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="trendingPush">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>270</y>
      <width>76</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Trending</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1051</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="discoverPush">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>180</y>
      <width>76</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);
alternate-background-color: rgb(31, 81, 255);</string>
    </property>
    <property name="text">
     <string>Discover</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="travelsPush">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>360</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Your Travels</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 81, 255);</string>
    </property>
    <property name="text">
     <string>NOMADIC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>45</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DASHBOARD</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>221</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 81, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>68</y>
      <width>81</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="logohead">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 81, 255);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Nomadic&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>26</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 81, 255);</string>
    </property>
    <property name="text">
     <string>UserName</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="accPush">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>460</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="credPush">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>115</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);
alternate-background-color: rgb(31, 81, 255);</string>
    </property>
    <property name="text">
     <string>Loyalty Creds</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>220</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>96</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DM Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>INSTANT BOOKINGS</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>81</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>220</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>96</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DM Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>LIVE LOCATION</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>81</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>96</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DM Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>TRAVEL HISTORY</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>81</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>96</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DM Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>NOMAD BOARD</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>81</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icon01.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icon04.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icon02.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icon03.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutPush">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 33, 36);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>652</y>
      <width>55</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>line</zorder>
   <zorder>logohead</zorder>
   <zorder>pushButton</zorder>
   <zorder>accPush</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>discoverPush</zorder>
   <zorder>trendingPush</zorder>
   <zorder>travelsPush</zorder>
   <zorder>credPush</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_5</zorder>
   <zorder>graphicsView_6</zorder>
   <zorder>graphicsView_7</zorder>
   <zorder>logoutPush</zorder>
   <zorder>line_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
